Create ui-c8y-1021-20-0-migrate-translation-ui-from-dtm-7751.md (#2708)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-20-0-migrate-translation-ui-from-dtm-7751.md

---
date: ""

> title: migrate translation UI from DTM (#7751)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Migrate translation user interface from DTM to new framework
```

Reviewer: ```suggestion
title: Migrate localization user interface from DTM application to Administration application
```

> title: Migrate localization user interface from DTM application to Administration application

Reviewer 2: ```suggestion
title: Migrated localization user interface from Digital Twin Manager application to Administration application
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:

>   - value: component-0UgqXH1Ys

Reviewer 2: ```suggestion
  - value: component-Tl88RYb4A
```

>     label: Administration

Reviewer 2: @reey Since this was previously in the DTM application, shouldn´t the label better be DTM?

>     label: Administration

Reviewer 2: ```suggestion
    label: Digital Twin Manager
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61483
version: 1021.20.0
---

> migrate translation UI from DTM (#7751)

Author: The Claude AI suggested to use the following body:
```suggestion
As part of the ongoing migration efforts, the user interface for managing translations has been migrated from the legacy DTM framework to a new modern framework. The functionality remains the same, allowing users to view, edit and manage translations as before. However, the underlying technology has been updated, providing better performance, stability and maintainability of the translations user interface going forward.
```

> As part of the ongoing migration efforts, the user interface for managing translations has been migrated from the legacy DTM framework to a new modern framework. The functionality remains the same, allowing users to view, edit and manage translations as before. However, the underlying technology has been updated, providing better performance, stability and maintainability of the translations user interface going forward.

Reviewer 2: ```suggestion
As part of the ongoing migration efforts, the user interface for managing translations has been migrated from the legacy Digital Twin Manager framework to a new modern framework. The functionality remains the same, allowing users to view, edit and manage translations as before. However, the underlying technology has been updated, providing better performance, stability and maintainability of the translations user interface going forward.
```
